### PR TITLE
Use ReactJS instead of Vue.js

### PR DESCRIPTION
**Pull request by kerphi**

### CODE REVIEW

File: ezmaster-front/public/fakeapi/config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "EZMASTER_PUBLIC_DOMAIN": "mywebsite.fr",
+  "EZMASTER_HOME_TITLE": "EzMaster",
+  "EZMASTER_HOME_DESCRIPTION": "EzMaster is a none technical backoffice tools for none IT administrators. It aims to manage applications and instances. One instance of an application is a docker packaged software having it's own config and it's own data. Each instances are isolated and independant, they can be stopped <i class=\"fa fa-stop-circle\"></i> or started <i class=\"fa fa-play-circle\"></i> easily on demand. Applications are \"ezmasterized\" docker images, they can be easiely downloaded <i class=\"fa fa-download\"></i> from dockerhub."
```
Comment (parmentf) [inline ezmaster-front/public/fakeapi/config.json]: I would have said "EzMaster is a **non**-technical..." (without `e` to `none`).

Comment (parmentf) [reply ezmaster-front/public/fakeapi/config.json]: But it's a fake API configuration, that's not very important :P

File: ezmaster-front/src/Home.js [hunk 2]
```diff
@@ -38,18 +38,20 @@ class Home extends Component {
...
+              dangerouslySetInnerHTML={{
+                __html: this.props.config.EZMASTER_HOME_TITLE || "EzMaster"
+              }}
+            />
+            <p
+              className="lead"
+              dangerouslySetInnerHTML={{
+                __html:
+                  this.props.config.EZMASER_HOME_DESCRIPTION ||
+                  'EzMaster is a none technical backoffice tools for none IT administrators. It aims to manage applications and instances. One instance of an application is a docker packaged software having it\'s own config and it\'s own data. Each instances are isolated and independant, they can be stopped <i class="fa fa-stop-circle"></i> or started <i class="fa fa-play-circle"></i> easily on demand. Applications are "ezmasterized" docker images, they can be easiely downloaded <i class="fa fa-download"></i> from dockerhub.'
```
Comment (kerphi) [inline ezmaster-front/src/Home.js]: thank you @parmentf look at this line, this is the default value if config.json is not yet loaded thanks to AJAX
I will change to non-technical here :-)

**Comment by kerphi** [comment]
@parmentf is it correct "for none IT administrators" ? or is "for non-IT administrators" better ?

**Comment by parmentf** [comment]
@kerphi I'd say "for non-IT administrators".

**Pull request merged**